### PR TITLE
Clarify statement of purpose in readme

### PR DESCRIPTION
**Pull request by jnqnfe**
Per discussion in [issue #18](https://github.com/JohnTitor/ctest2/issues/18)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 [dox]: https://docs.rs/ctest2
 
-**Note: This is a fork repository and we intend to use this in libc-test only.**
+**Note: This is a fork of [`ctest`], intended as a temporary replacement until maintenance of [`ctest`] resumes or ownership is transferred.**
```
Comment (JohnTitor) [inline README.md:7]:
```suggestion
**Note: This is a fork of [`ctest`], intended as a temporary replacement until maintenance of [`ctest`] resumes.**
```

I don't expect the latter.

Comment (JohnTitor) [reply README.md:7]: Also, this doesn't mention the use in libc.

Comment (jnqnfe) [reply README.md]: Ok, force-pushed suggested amendment.

**Comment by JohnTitor** [comment]
Thanks!

**Pull request merged**